packages/tools/pluggable-widgets-tools/configs/rollup.config.native.js

```diff
@@ -62,7 +66,48 @@ export default async args => {
                 }),
                 ...(i === 0 ? getClientComponentPlugins() : []),
                 json(),
-                collectDependencies({ outputDir: outDir, onlyNative: true, widgetName }),
+                collectDependencies({
+                    outputDir: outDir,
+                    onlyNative: true,
+                    widgetName,
+                    ...(production && i === 0
+                        ? {
+                              licenseOptions: {
+                                  thirdParty: {
+                                      output: [
+                                          {
+                                              file: join(outDir, "dependencies.txt")
+                                          },
+                                          {
+                                              file: join(outDir, "dependencies.json"),
+                                              template: dependencies =>
+                                                  JSON.stringify(
+                                                      dependencies.map(dependency => ({
+                                                          [dependency.name]: {
+                                                              version: dependency.version,
+                                                              type: "test",
+                                                              transitive: false,
+                                                              url: dependency.homepage
+                                                          }
+                                                      }))
+                                                  )
+                                          }
+                                      ]
+                                  },
+                                  ...(mxLicensePath
```

Comment:
I don't understand the purpose of this. Is this for when a LICENSE file doesn't exist in the widget's sourcecode `src/` folder, but elsewhere?

Reply:
Removed that part